packages/frontend/app/components/program-year/leadership-expanded.hbs

```diff
@@ -1,33 +1,42 @@
 <div
   class="program-year-leadership-expanded"
   data-test-program-year-leadership-expanded
-  ...attributes
+  {{did-insert (perform this.load) @programYear}}
 >
+{{#if this.load.isRunning}}
```

Comment:
I think instead of taking a step back into modifiers we could just observe `this.programYearDirectors.isResolved` here and get the same behavior without any other change.

Reply:
@jrjohnson i tried the approach where is checked the state of the Resource in the template as you described,  and it does not eliminate the flash of content.

to your point - going back to modifiers feels like a regression, so this is a bit of a knee-jerk solution to the problem. on the other hand - this works. 

i can take this back to the bench to see if i can make this work with Resources somehow. let me know, thanks.

Reply:
Going to merge as is and @stopfstedt will follow up.